sig-scalability/slos/slos.md

```diff
@@ -123,9 +124,12 @@ if the request is about a single object, (b) `namespace` if it is about objects
 from a single namespace or (c) `cluster` if it spawns objects from multiple
 namespaces.
 
+<a name="footnode6">[6\]</a>A `stateless pod` is defined as a pod that doesn't
```

Comment:
Actually, IIUC even with just secrets and configmaps it might be still possible to affect the SLI. For e.g if there are too many or too big of them.

Reply:
For both - we may tune it if/when we have numbers. I don't think we can say more now.

Reply:
Sgtm - let's not complicate this already too-complicated SLI definition for now :)